spec/support/components/work_packages/filters.rb

```diff
@@ -148,8 +148,8 @@ def expect_no_filter_by(name, selector = nil)
         id = selector || name.downcase
 
         retry_block do
-          page.raise_if_found_select("operators-#{id}")
-          page.raise_if_found_select("values-#{id}")
+          expect(page).not_to have_select("operators-#{id}")
```

Comment:
This will cause the retry_block to become irrelevant, as expect() will not raise an error, but mark the spec as failing internally